assets/queries/openAPI/security_operations_empty_array/metadata.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "663c442d-f918-4f62-b096-0bf5dcbeb655",
+  "queryName": "Operations Security Field should not have an empty array",
```

Comment:
The folder name is different from the 'queryName' in the metadata file
Maybe it would be better if every word of 'queryName' starts with uppercase